src/Jira/Api.php

```diff
@@ -300,7 +300,7 @@ public function getVersions($projectKey)
      *
      * @return mixed
      */
-    public function getPriorties()
+    public function getPriorities()
```

Comment:
Please also add `getPriorties` (old method name) that would:

* trigger `E_USER_DEPRECATED` notice (lets people know, that new method should be called instead)
* call `getPriorities` (new method name) internally (to prevent BC break)